common/persistence/src/test/java/app/coronawarn/server/common/persistence/service/TraceTimeIntervalWarningServiceTest.java

```diff
@@ -115,7 +89,7 @@ void testSortedRetrievalResult() {
             CheckinsDateSpecification.HOUR_SINCE_EPOCH_DERIVATION
                 .apply(Instant.now().getEpochSecond()) - 10);
 
-    List<CheckIn> checkins = new ArrayList<>(List.of(
+    List<CheckIn> checkins = new java.util.ArrayList<>(List.of(
```

Comment:
Why is this prefixed?
```suggestion
    List<CheckIn> checkins = new ArrayList<>(List.of(
```